linux/dir_tools.bsh

```diff
@@ -82,13 +82,14 @@ _VSI_TEMP_PATHS=()
 #
 # :Arguments: ``$1`` - Name of variable where temporary path will be stored
 #             [``$2``...] - Optionally add additional arguments to mktemp call. The most useful thing to add would be ``-t``, as this is in all the OSes. ``-d`` is often used to make a directory
+# :Parameters: ``DIR_TOOLS_KEEP_TEMP_PATHS`` - Set to keep the temporary paths instead of deleting them (typically for debugging). This can be used in two difference ways. It can be set on each call to :func:`make_temp_path`, to keep an individual path. It can also be set globally so that when bash exits, none of the paths will be removed. Default: unset.
```

Comment:
```suggestion
# :Parameters: ``DIR_TOOLS_KEEP_TEMP_PATHS`` - Set to keep the temporary paths instead of deleting them (typically for debugging). This can be used in two difference ways. 1) It can be set on each call to :func:`make_temp_path`. 2) It can also be set globally so that when bash exits, none of the paths will be removed. Default: unset.
```